.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+clang/ @erichkeane, @Fznamznon
+
+clang/include/clang/Driver/ @mdtoguchi @AGindinson
+clang/lib/Driver/ @mdtoguchi @AGindinson
+
+llvm-spirv/ @AlexeySotkin, @AlexeySachkov
+
+opencl-aot/ @dm-vodopyanov
+
+libdevice/ @asavonic, @vzakhari
+
+sycl/doc/extensions/ @mkinsner, @jbrodman
+
+sycl/doc/ @pvchupin, @kbobrovs
+
+sycl/ @romanovvlad
```

Comment:
We need second owner here. @romanovvlad  please suggest second owner

Reply:
@bader